chpl/chpl-service/src/main/java/gov/healthit/chpl/questionableactivity/CertificationResultQuestionableActivityProvider.java

```diff
@@ -1,4 +1,4 @@
-package gov.healthit.chpl.questionableActivity;
+package gov.healthit.chpl.questionableactivity;
```

Comment:
why remove the camel case?

Reply:
Package names should be all lower case - https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html

Reply:
Oh did not know that!